api/src/main/java/io/opentelemetry/trace/TracerProvider.java

```diff
@@ -26,7 +26,7 @@
  * @since 0.1.0
  */
 @ThreadSafe
-public interface TracerRegistry {
+public interface TracerProvider {
```

Comment:
Same comment about provider not being a java provider?

Reply:
Added in 73ed199